inspirehep/modules/disambiguation/core/ml/utils.py

```diff
@@ -0,0 +1,330 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction
+
+"""Disambiguation core ML utils."""
+
+from __future__ import absolute_import, division, print_function
+
+from beard.utils import (
+    given_name,
+    given_name_initial,
+    name_initials,
+    normalize_name,
+)
+
+
+def get_author_full_name(s):
```

Comment:
one-letter parameters are bad. What is `s`?

Reply:
Should be signature. But yes, utils needs an overall refactoring.